### PR TITLE
5026: update cookie notice

### PR DESCRIPTION
**Pull request by chrisholmes**
the cookie notice must be changed to reflect the removal of the secure cookie.

@gdsamit has reviewed the changes and has given it an OK.

### CODE REVIEW

**Comment by robinwhittleton** [comment]
Looks fine, will merge when tests finish.

File: app/views/static/cookies.cy.html.erb [hunk 1]
```diff
@@ -34,18 +34,13 @@ end %>
...
-    </tr>
-    <tr>
       <td><var>x_govuk_session_cookie</var></td>
       <td>This cookie contains randomly generated characters so we can identify you during your visit</td>
       <td>When you close your browser window</td>
     </tr>
     <tr>
       <td><var>_verify-frontend_session</var></td>
-      <td>This cookie is used to hold information regarding your session, including when you started your session, which government service you’re using, your answers to filter questions, and which certified company you select so that we can present information relevant to your choices. This information is secured so that only GOV.UK Verify can read it</td>
+      <td>When you’re trying to use a service on GOV.UK, this cookie makes sure that no other user can login with your information. It is also used to hold information regarding your session, including when you started your session, which government service you’re using, your answers to filter questions, and which certified company you select so that we can present information relevant to your choices. This information is secured so that only GOV.UK Verify can read it</td>
```
Comment (willp-bl) [inline app/views/static/cookies.cy.html.erb]: GOV.UK -> GOV.UK Verify

Comment (chrisholmes) [inline app/views/static/cookies.cy.html.erb]: It's GOV.UK because the service in this case is the Relying Party

**Comment by gdsamit** [comment]
I think we need to say "log in" not "login" as we're using it as a verb not a noun.

**Comment by chrisholmes** [comment]
amended!

**Pull request merged**